scripts/dex-router/src/router/jupiter.ts

```diff
@@ -0,0 +1,104 @@
+import { Router, RouterOutput } from "../types";
+import { Connection, PublicKey, TransactionInstruction } from "@solana/web3.js";
+
+const jupiterBaseUrl = "https://quote-api.jup.ag/v6/";
+const jupiterQuoteUrl = new URL("quote", jupiterBaseUrl);
+const jupiterSwapIxsUrl = new URL("swap-instructions", jupiterBaseUrl);
+const maxAccounts = 30;
+
+export class JupiterRouter implements Router {
+  private chainId: string;
+  private connection: Connection;
```

Comment:
not used

Reply:
i designed the interface this way bc for other routers we may need to pass in a connection. for now i can remove it